src/room.ts

```diff
@@ -222,7 +222,7 @@ export class Room extends EventEmitter implements Sayable {
             tmpName = contact.alias() || contact.name()
             break
           case 'alias':
-            tmpName = member.DisplayName || contact.name()
+            tmpName = member.DisplayName
```

Comment:
Should update JSDoc at https://github.com/wechaty/wechaty/pull/303/files#diff-a260093009080ad4951308e3d5ea54fdR342